app.js

```diff
@@ -43,4 +46,12 @@ app.use(function (err, req, res, next) {
   res.render("error");
 });
 
+function checkAuth(req, res, next) {
+  if (req.isAuthenticated()) {
+    return next();
+  }
```

Comment:
You could leave this in just one line (it is just a comment but not completely necessary)
`if (req.isAuthenticated()) return next();`

Reply:
Actually i realized i don't use that function anymore :rofl: